src/main/js/src/components/Registration/ExamSessionList/ExamSessionListItem/ExamSessionListItem.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import moment from 'moment';
+import { withTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+import classes from './ExamSessionListItem.module.css';
+import { nowBetweenDates, levelDescription } from '../../../../util/util';
+import {
+  DATE_FORMAT,
+  DATE_FORMAT_WITHOUT_YEAR,
+} from '../../../../common/Constants';
+
+const examSessionListItem = ({ examSession: session, language, t }) => {
+  console.log(session);
+
+  const date = (
+    <div className={classes.Date}>
+      {moment(session.session_date).format(DATE_FORMAT)}
+    </div>
+  );
+  const exam = (
+    <div className={classes.Exam}>
+      <strong>{`${language.name}, ${levelDescription(
+        session.level_code,
+      ).toLowerCase()}`}</strong>
+    </div>
+  );
+  const name = session.location[0].name;
```

Comment:
Exam session location is currently stored in all supported languages and can be shown in user language.